src/relay/backend/graph_runtime_codegen.cc

```diff
@@ -254,8 +254,6 @@ class GraphRuntimeCodegen : public backend::MemoizedExprTranslator<std::vector<G
 
     ret.lowered_funcs = lowered_module.per_target_module;
     std::cout << "Modules: " << ret.lowered_funcs << std::endl;
```

Comment:
please removed this printing msg

Reply:
Hey @zhiics, thanks for checking this branch out! At the moment this branch is just a PoC @jroesch and I are working on. There's lots of cleanup we plan to batch together down the line a bit.

Reply:
oops.  I didn't notice the branch... Thanks for working on it